app/src/sharedTest/java/org/oppia/app/profile/ProfileChooserFragmentTest.kt

```diff
@@ -123,12 +130,22 @@ class ProfileChooserFragmentTest {
 
   @Test
   fun testProfileChooserFragment_clickAddProfile_checkOpensAdminAuthActivity() {
+    profileTestHelper.initializeProfiles()
     ActivityScenario.launch(ProfileActivity::class.java).use {
       onView(atPosition(R.id.profile_recycler_view, 2)).perform(click())
       intended(hasComponent(AdminAuthActivity::class.java.name))
     }
   }
 
+  @Test
+  fun testProfileChooserFragment_clickAdminProfileWithNoPin_checkOpensAdminPinActivity() {
+    profileManagementController.addProfile("Sean", "", null, true, R.color.abc_tint_default, true)
```

Comment:
Suggest also using named arguments for the null & true parameter here (you can use them for all of the parameters to make it really explicit).